### ISSUE TITLE
Question : Integration with bitbucket server

### ISSUE THREAD

**Issue by unknown1234** [buildbot/buildbot_travis#99]
Hi buildbot team!

Is it possible to integrate buildbot_travis with bitbucket server ?
In order to trigger build with webhooks and get the results back in bitbucket ?

Thanks

Regards, Cyril

**Comment by seankelly** [comment]
Yes, but I had to subclass `BitbucketServerStatusPush`. First was to configure it with the desired builds to report back set using the `builders` parameter. Second, subclass the `filterBuilds` method to check if the build's `buildername` property was in `self.builders`.
